src/components/app/ProfileViewer.css

```diff
@@ -43,3 +43,15 @@
 .profileViewerZipButton:hover {
   border-color: #000;
 }
+
+.profileViewerSplitter {
+  /* Reset the position: absolute that is set by the splitter-layout class */
+  position: unset;
+}
+
+.profileViewerSplitter .layout-pane:not(.layout-pane-primary) {
+  display: flex;
+  overflow: hidden;
+  max-height: var(--profile-viewer-splitter-max-height);
+  flex-direction: column;
+}
```

Comment:
should we have a min-height that would be the size of one track + the 2 bars ? Or maybe just the 2 bars ?

Reply:
I thought about this, but I kind of like the fact that I can totally maximize the space for the chart. Happy to follow-up with this if other folks have opinions too.